<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWeldWidget</class>
 <widget class="QWidget" name="MainWeldWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWeldWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#weldLabel{
background-color:rgb(0, 130, 195);
color: white;   
font: 26pt;
font-weight:bold;
border-style: outset;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="weldLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Weld</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="leftVerticalLayout" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="Ruler" name="leftRuler" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="emptyWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="centerVerticalLayout" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="weldVerticalLayout"/>
       </item>
       <item>
        <widget class="Ruler" name="bottomRuler" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rightVerticalLayout" stretch="1,1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="WeldPropertiesWidget" name="weldProperties" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="StaticWeldWidget" name="staticWeld" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="btnHorizontalLayout">
         <item>
          <widget class="QPushButton" name="backBtn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="finishedBtn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Finished</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StaticWeldWidget</class>
   <extends>QWidget</extends>
   <header>static_weld_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeldPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>weld_properties_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Ruler</class>
   <extends>QWidget</extends>
   <header>ui/display/ruler/ruler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
